lib/senkyoshi/models/file_resource.rb

```diff
@@ -19,5 +19,19 @@ def self.from(xml, pre_data, _resource_xids = nil)
     def iterate_xml(_xml, _pre_data)
       self
     end
+
+    def create_module(course)
+      course.canvas_modules ||= []
+      cc_module = Course.master_module(course)
```

Comment:
Why not have the course manage the modules? You wouldn't have to do the line above `course.canvas_modules ||= []` as `course` would already be aware of its `canvas_modules`

I mean you have dryed it up with the Class method. But I think an instance method would be better. Not a requirement, but my opinion. :]